kolibri/plugins/coach/assets/src/app.js

```diff
@@ -196,10 +196,10 @@ class CoachToolsModule extends KolibriModule {
         },
         {
           name: PageNames.EXERCISE_RENDER,
-          path: '/:userId/:contentId/exercise-render',
+          path: '/:classId/exercise-render/:userId/:channelId/item/:contentId',
```

Comment:
Actually, this URL is going to become moot! Each "tab" in reports is going to call the action separately